compiler/src/iree/compiler/Codegen/SPIRV/test/tile_and_vectorize_conv.mlir

```diff
@@ -470,14 +470,8 @@ hal.executable private @nchw_conv_static_shape_f32 {
 //      CHECK:     scf.for %{{.*}} = %c0 to %c3 step %c1
 // CHECK-SAME:         -> (tensor<2x8x1x4xf32>)
 
-// TODO: remove these vector.extract & vector.insert pairs due to vector.transpose
-// CHECK-COUNT-96: vector.insert %{{.+}}
```

Comment:
would it make sense to add a CHECK-NOT to make sure it doesn’t regress?

Reply:
The newly added test for have `CHECK-NOT` to guard against it.